src/app/pages/ui/components/slim/slim.html

```diff
@@ -0,0 +1,28 @@
+<div class="widgets">
+    <div class="row">
+        <div class="col-md-12">
+            <ba-card title="Slim loading bar" class="slim-loading-bar-block">
+                <button class="btn btn-default" (click)="setProgres30()">Set progress equals 30</button>
+                <br>
+                <button class="btn btn-default" (click)="incrementProgress()">Increment progress</button>
+                <br>
```

Comment:
Looks good. Also, could you update the hot-table styles like ng2-smart-table (transparent background, colors and etc.)?